swim/node_bootstrap_test.go

```diff
@@ -74,21 +74,20 @@ func (s *BootstrapTestSuite) TestBootstrapJoinsTimeOut() {
 }
 
 func (s *BootstrapTestSuite) TestBootstrapDestroy() {
-	var lerr lError
+	errChan := make(chan error)
 
 	go func() {
 		_, err := s.node.Bootstrap(&BootstrapOptions{
 			Hosts:       fakeHostPorts(1, 1, 1, 10),
 			JoinTimeout: time.Millisecond,
 		})
-		lerr.Set(err)
+		errChan <- err
 	}()
 
-	time.Sleep(2 * time.Millisecond)
 	s.node.Destroy()
```

Comment:
This is a good question. In theory, it's possible for the Bootstrap to be scheduled before the continuation of the test function. I've heard about some guarantees around goroutine scheduling. I'll try to dig up an answer from somewhere and make change if I have to.